Description: Add the AM_PROG_AR macro to configure.ac to avoid automake
 warnings in ltlibrary.am while processing the Libtool library 'libexif.la'
Author: Hugh McMaster <hugh.mcmaster@outlook.com>
Last-Update: 2018-03-14

Index: libexif-0.6.21/configure.ac
===================================================================
--- libexif-0.6.21.orig/configure.ac
+++ libexif-0.6.21/configure.ac
@@ -70,6 +70,7 @@ AC_SUBST([LIBEXIF_CURRENT_MIN],[`expr $L
 LIBEXIF_VERSION_INFO="$LIBEXIF_CURRENT:$LIBEXIF_REVISION:$LIBEXIF_AGE"
 AC_SUBST([LIBEXIF_VERSION_INFO])
 
+AM_PROG_AR
 AC_PROG_CC
 AC_C_CONST
 AC_C_INLINE
